src/core/dset.c

```diff
@@ -595,6 +598,17 @@ static int print_contact_str(char **dest, str *uri, qvalue_t q, str *path, char
 		memcpy(p, buf.s, buf.len);
 		p += buf.len;
 	}
+
+	/* branch flags (not SIP standard conformant) */
+	buf.s = int2str(flags, &buf.len);
+	if (p + FLAGS_PARAM_LEN + buf.len > end) {
+		return -1;
+	}
+	memcpy(p, FLAGS_PARAM, FLAGS_PARAM_LEN);
+	p += FLAGS_PARAM_LEN;
+	memcpy(p, buf.s, buf.len);
+	p += buf.len;
```

Comment:
This should be made somehow optional, not being standard, as well as eventually do not add the flags parameter if the value is 0 (not set).

Reply:
Patch 4 makes this optional.

Reply:
Seems that follow up patches add the option. For the future, when doing the PR merge the patches done to the same component in one, makes the review simpler as one does it in the order they are listed and later discovers is no longer same code, being again changed.